aci/resource_aci_bgppeerp.go

```diff
@@ -633,28 +639,33 @@ func resourceAciBgpPeerConnectivityProfileUpdate(ctx context.Context, d *schema.
 			bgpLocalAsnPAttr.LocalAsn = LocalAsn.(string)
 		}
 
-		bgpLocalAsnP := models.NewLocalAutonomousSystemProfile(fmt.Sprintf("localasn"), PeerConnectivityProfileDn, desc, bgpLocalAsnPAttr)
-		bgpLocalAsnP.Status = "modified"
+		if _, ok := d.GetOk("local_asn"); ok {
```

Comment:
you can probably merge those two if's?
But isn't there a case for local_asn_propagate to be set when local_asn is not set?

Reply:
Yes, we can merge the two.

No, local_asn needs to be set in order to set local_asn_propagate to any value in TF (because of the flag RequiredWith:local_asn)

I checked in the UI and though propagate can be set to any value without setting asn, localasn is only created when asn is set.